src/apel_plugin.py

```diff
@@ -145,22 +135,57 @@ def get_submit_host(record, config):
             "%2F", "/"
         )
     except TypeError:
-        logging.warning(
-            f"No submithost found in record {record.record_id}, using default"
-        )
+        logging.warning(f"No submithost found in record {record.record_id}")
+        logging.warning(f"Send default {default_submit_host}")
         submit_host = default_submit_host
 
     return submit_host
 
 
+def get_voms_info(record, config):
+    meta_key_voms = config["auditor"].get("meta_key_voms")
+    voms_dict = {}
+
+    try:
+        voms_string = record.meta.get(meta_key_voms)[0].replace("%2F", "/")
```

Comment:
You may want to create a function which does the replacement of the characters. The replacement is done in multiple places. If we have to exchange other characters as well, then you only need to change it in one place.